benchmarks/core/braille.bril

```diff
@@ -0,0 +1,249 @@
+# converts the input integer into a braille code.
+#
+# Example
+# - input: 408
+# - output:
+#   701110110
+#   701011111
+#   711000000
+#   (7 is a convention; a header to print leading zeros)
+#
+# * Braille code for the numeric indicator:
+#   (placed only once before each number)
+# 01
+# 01
+# 11
+#
+# * Braille codes for 
+#   digits 0, 1, ... 9:
+# 1   2   3   4   5   6   7   8   9   0
+# 10  10  11  11  10  11  11  10  01  01
+# 00  10  00  01  01  10  11  11  10  11
+# 00  00  00  00  00  00  00  00  00  00
+
+
+@braille_in_printable_int(input: int): int {
+	one: int = const 1;
+	two: int = const 2;
+	three: int = const 3;
+	four: int = const 4;
+	five: int = const 5;
+	six: int = const 6;
+	seven: int = const 7;
+	eight: int = const 8;
+	nine: int = const 9;
+	ten: int = const 10;
+	quotient: int = div input ten;
+	tmp: int = mul ten quotient;
+	last_digit: int = sub input tmp;
+	
+}
```

Comment:
I was updating my bril-to-llvm translator and it currently errors on this. Since the return type is int, but nothing is returned, I believe this is ill-typed(though apparently not caught by brilck).

Part of that might be because this code is dead?

Reply:
Good point. I'm going to just delete this dead code, and then think on the other thread about how to check this statically!